withdraw-by-check/LETTERSPECS/BANNO.CHECK.WITHDRAW.V1.CFG

```diff
@@ -49,6 +76,7 @@ CA:
 *  Enter account level warning codes to exclude check withdrawals from
 *  any share or loan.  Separated by commas only and formatted to three
```

Comment:
What does "Separated by commas only" mean?  Is this one of our standard range/list parameters or is it handled differently in the code?  Can you have a range separated by dash?  Same comment for each of these warning code parameters.